scripts/valhalla_build_config

```diff
@@ -98,7 +98,8 @@ config = {
     },
     'max_reserved_labels_count': 1000000,
     'clear_reserved_memory': False,
-    'extended_search': False
+    'extended_search': False,
+    'invariant_recosting': True
```

Comment:
i think this should be decided at runtime, not via config. simply add a new field to the request structure that tells valhalla to use bidirectional a* even if the date_time type isnt invariant and  you're basically done (other than the actual work of doing the linear combination in GetSpeed and passing the seconds_from_now in the costing function)

Reply:
sorry maybe a little more info, you'd need to account for this in thor/route_action where we select the algorithm to use.